spec/components/header_component/header_component_preview.rb

```diff
@@ -1,5 +1,5 @@
 class HeaderComponent::HeaderComponentPreview < ViewComponent::Preview
-  layout "component_preview"
+  layout "base"
```

Comment:
I think it would be worth to use this layout as a default and not just for this component. 

https://github.com/alphagov/forms-admin/blob/main/app/controllers/component_preview_controller.rb#L5

Reply:
Done!